lms/static/scripts/frontend_apps/components/LMSFilePicker.tsx

```diff
@@ -75,6 +75,11 @@ type LMSFilePickerProps = {
    * Render path breadcrumbs and allow sub-folder navigation?
    */
   withBreadcrumbs?: boolean;
+
+  /**
+   * An alternative title for the modal dialog. Defaults to "Select file"
+   */
+  title?: string;
```

Comment:
There are a few other UI strings that need to be adapted. For English, it would be adequate to provide singular and plural forms of the resource name (`file`, `files`, `page`, `pages`) and substitute those into the messages. In future we might have to provide alternative versions of the whole message to be grammatically correct.

Reply:
See my comment here https://github.com/hypothesis/lms/pull/5717#discussion_r1347090485

Reply:
As [discussed on slack](https://hypothes-is.slack.com/archives/C1M8NH76X/p1696593792109319), I have refactored the `LMSFilePicker` to expose a prop that tells the typo of documents it handles.

Using that prop, all user-facing messages can be dynamically composed.